queries.sql

```diff
@@ -14,4 +14,101 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+
+-- Begin transaction
+BEGIN;
+
+-- Update animals table by setting species to unspecified
+UPDATE animals SET species = 'unspecified';
+
+-- Verify change
+SELECT * FROM animals;
+
+-- Rollback transaction
+ROLLBACK;
+
+-- Verify that species column went back to the state before the transaction
+SELECT * FROM animals;
+
+
+-- Begin transaction
+BEGIN;
+
+-- Update animals table by setting species to digimon for all animals whose name ends in mon
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Update animals table by setting species to pokemon for all animals that don't have a species already set
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+
+-- Commit transaction
+COMMIT;
```

Comment:
- [ ] Before we can commit here we have to verify that the change was made using a `select statement`, kindly do so like you have done after the commit.
![image](https://user-images.githubusercontent.com/81932478/228373672-acdc8ef3-226d-4158-8389-a9d04a04f245.png)